example/CMakeLists.txt

```diff
@@ -51,5 +51,6 @@ add_custom_command(TARGET ${PROJECT_NAME} POST_BUILD
 )
 
 get_filename_component(PROJECT_ROOT ${CMAKE_CURRENT_LIST_DIR}/.. ABSOLUTE)
+set(STUB_COMPILE_DEFS "STUB_LOG_ENABLED;STUB_LIB_LOG_BUF")
```

Comment:
if we decided to use a such variable to pass configs to esp-stub-lib, lets rename it to a kind of `ESP_STUB_LIB_DEFS`